04.class/memosData.js

```diff
@@ -0,0 +1,39 @@
+import { promises as fs } from "fs";
+
+class MemosData {
+  constructor() {
+    this.memoFilePath = "data/memos.json";
+  }
+
+  async read() {
+    try {
+      const memoData = await fs.readFile(this.memoFilePath, "utf-8");
+      return JSON.parse(memoData);
+    } catch (error) {
+      if (error.code) {
+        if (error.code === "ENOENT") {
```

Comment:
`code` がない場合は `undefined` との比較が行われるだけで、このコードと元のコードは挙動が変わらないはずなので、元のコードのままでよいですー。

```suggestion
      if (error.code === "ENOENT") {
```

Reply:
元のコードに戻しました！